lib/src/fnc/script/classes/response.rs

```diff
@@ -0,0 +1,140 @@
+#[js::bind(object, public)]
+#[quickjs(bare)]
+#[allow(non_snake_case)]
+#[allow(clippy::module_inception)]
+pub mod response {
+	use crate::fnc::script::classes::headers::headers::Headers;
+	use crate::sql::{json, Value};
+	use futures::lock::Mutex;
+	use js::Result;
+	use std::sync::{atomic::AtomicBool, atomic::Ordering, Arc};
+
+	use super::JsArrayBuffer;
+
+	#[quickjs(cloneable)]
+	#[derive(Clone)]
+	pub struct Response {
+		#[quickjs(skip)]
+		pub(crate) inner: Arc<Mutex<surf::Response>>,
+		headers: Headers,
+		#[quickjs(hide)]
+		statusCode: surf::http::StatusCode,
+		#[quickjs(hide)]
+		bodyUsed: Arc<AtomicBool>,
+	}
+
```

Comment:
Could we add the following function to this class?

```rs
Could we add the following function to this class?

```rs
// Convert the object to a string
pub fn toString(&self) -> String {
		String::from("[object Response]")
}
```